apps/console/src/features/applications/components/settings/access-configuration.tsx

```diff
@@ -178,11 +181,12 @@ export const AccessConfiguration: FunctionComponent<AccessConfigurationPropsInte
     const { t } = useTranslation();
     const { getLink } = useDocumentation();
 
-    const dispatch = useDispatch();
+    const dispatch: Dispatch<any> = useDispatch();
```

Comment:
Shall we just use `Dispatch`?

Reply:
Resolved by [7aa7c4d](https://github.com/wso2/identity-apps/pull/3650/commits/7aa7c4d7a93be99ecd1f614e46cc75fe030ec640)